Sources/Howmuch/Utilities/HeaderAttributesBuilder.swift

```
@@ -0,0 +1,17 @@
+struct HeaderAttributesBuilder {
+    private typealias Keys = Constants.Request.Header
+    private var addedHeaders: [HeaderAttribute] = []
+
+    @discardableResult
+    mutating func addApiKey(currencyExchangeRepository: CurrencyExchangeRepositoryType) -> Bool {
```

Can you throw an error instead of returning a boolean?

---

I refactor the func to throw error in this [commit](https://github.com/ts-robih-robihamanto/howmuch/pull/1/commits/51d74ae1c3e8431289f28742b93972041cddf59b).